apps/wallet/src/ui/app/components/logo/index.tsx

```diff
@@ -1,27 +1,44 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import classnames from 'classnames/bind';
+import cl from 'classnames';
 
+import { API_ENV } from '../../ApiProvider';
+import { Text } from '../../shared/text';
 import Icon, { SuiIcons } from '_components/icon';
 
-import st from './Logo.module.scss';
-
-const cl = classnames.bind(st);
+const networkNames: Record<API_ENV, string> = {
+    [API_ENV.local]: 'Local',
+    [API_ENV.testNet]: 'Testnet',
+    [API_ENV.devNet]: 'Devnet',
+    [API_ENV.customRPC]: 'Custom RPC',
+};
 
 type LogoProps = {
-    size?: 'normal' | 'big' | 'bigger' | 'huge';
-    txt?: boolean;
```

Comment:
I assume we don't have any use-cases for size and txt anymore?

Reply:
yea, no use-cases in code. i'll double check the figma just to make sure we won't need to support it

Reply:
double-checked figma as well - don't see a use case for either `txt` or `size`